packaging/microshift/config.yaml

```diff
@@ -29,3 +29,20 @@ apiServer:
 debugging:
   # Log verbosity ('Normal', 'Debug', 'Trace', 'TraceAll'):
   #logLevel: 'Normal'
+
+etcd:
+  # The limit on the size of the etcd database, in bytes; thee etcd server will start failing writes if its size on disk reaches this value. (Default: 2GB)
+	#quotaBackendBytes: 2147483648
+
+	# How often, in nanoseconds, to check the conditions for defragmenting the etcd database (0 means no defrags, except for a single on startup if `doStartupDefrag` is set). (Default: 5 minutes)
```

Comment:
Same here. Do we expect someone to use nanoseconds? We ought to be able to use at least seconds.